### DISCUSSION TITLE
Detailed Installation Instructions

### DISCUSSION THREAD

**Discussion by AfricanWarrior254** [orbat-mapper/orbat-mapper#94] [Q&A] [answered]
Hello there, 

I've come across this ypur project and I'm dying to make use of the OrBat chart editor but unfortunately I'm not too savvy with GitHub and apart from the clone step, I'm finding it hard to follow the rest of the process.

Any help will be appreciated.

**Comment by kjellmf** [comment, marked as answer]
The installation instructions are indeed minimal. Sorry about that. I will improve it when orbat-mapper is more mature. The current instructions are intended for front end developers familiar with standard node.js-based tooling. If you are unfamiliar with npm you can find many guides online, for instance this one https://www.sitepoint.com/npm-guide/

Please note that orbat-mapper is a work in progress and the ORBAT chart functionality is currently limited. The preview version at https://preview--orbat-mapper.netlify.app/ is basically the same as the one you'll get if you install orbat-mapper locally.

**Reply by kjellmf** [reply]
Updated the README with a missing step and a link to the vite documentaiton  [021f047](https://github.com/kjellmf/orbat-mapper/commit/021f0475690853cbe4c7e8a4ba240f10f0d475e0).

**Reply by kjellmf** [reply]
Also added a section about offline usage.
